api/cli/cli.go

```diff
@@ -16,8 +16,8 @@ import (
 type Service struct{}
 
 const (
-	errInvalidEndpointProtocol       = portainer.Error("Invalid endpoint protocol: Portainer only supports unix:// or tcp://")
-	errSocketNotFound                = portainer.Error("Unable to locate Unix socket")
+	errInvalidEndpointProtocol       = portainer.Error("Invalid endpoint protocol: Portainer only supports unix://, npipe:// or tcp://")
+	errSocketNotFound                = portainer.Error("Unable to locate filesystem socket")
```

Comment:
Can you rename `errSocketNotFound` to `errSocketOrNamedPipeNotFound` and replace the message by: `Unable to locate Unix socket or named pipe`